runbot_travis2docker/models/runbot_branch.py

```diff
@@ -41,7 +41,8 @@ def _ssh_keyscan(self, ssh):
         The question is like to:
             'Are you sure you want to continue connecting (yes/no)?'"""
         cmd = ['ssh-keyscan', '-p']
-        match = re.search(r'@([^/]+(?=:|/]))?(?::)?([^/]+)?/', ssh)
+        match = re.search(r'@(?P<host>[^/]+(?=:|/]))?(?::)?(?P<port>[^/]+)?/',
```

Comment:
@moylop260 
I try to use the expression regular existing within the function "get_repo_data" but this not work as should be

![image](https://user-images.githubusercontent.com/1387970/30464124-a7961d2a-999d-11e7-8824-d38983461a5d.png)

I add group name to the regular expression existing in this code

![image](https://user-images.githubusercontent.com/1387970/30464001-fd1d232a-999c-11e7-859a-6016cbefecb4.png)

Only one trick the function "sorted" is required to get first the port and the second one the host